### DISCUSSION TITLE
Send and receive messages to Bing AI Copilot

### DISCUSSION THREAD

**Discussion by sjm008** [Terrance/SkPy#239] [Q&A] [answered]
very great tool. I really enjoyed using your library to communicate with Skype. However, when I tried to send message to Bing using your library it did not work. I checked the Sk?ype ID of Bing in my software but when I send a message, it sends it to that ID but not Bing. Do you know if there is a way to solve this

**Comment by Terrance** [comment, marked as answer]
See the discussion in #225 for how to retrieve the Bing user as a bot.  In short, Skype for Web does not advertise it as a bot in the usual way, which means trying to fetch it by ID alone will result in the wrong type of user being identified -- you can use [`SkypeContacts.bot`](https://skpy.t.allofti.me/reference/api.html#skpy.user.SkypeContacts.bot) **before** syncing contacts (or after clearing the cache) to ensure it's considered a bot.
